e2e-tests/portal/cypress/e2e/journeys/payment_officer/previous-reports/list-previous-reports.spec.js

```diff
@@ -0,0 +1,63 @@
+const { previousReports } = require('../../../pages');
+const MOCK_USERS = require('../../../../fixtures/users');
+const relativeURL = require('../../../relativeURL');
+
+// TODO FN-980 update to payment officer role
+const { BANK1_MAKER1 } = MOCK_USERS;
+
+context('List previous utilisation reports', () => {
```

Comment:
it's a bit of an obvious one but can we have a test that when you first load the page it defaults to the most recent year that has reports please?

Reply:
Yes, I've added a test, let me know what you think as I was trying to test without using knowledge of the test data, if that makes sense?